spyvm/primitives.py

```diff
@@ -787,6 +787,14 @@ def func(interp, s_frame, w_rcvr):
     # TODO: figure out whether we should decide the width an report it in the SCREEN_SIZE primitive
```

Comment:
Maybe this change actually fixes that TODO? If we agree that it does, remove it.